app/templates/public/sessions.hbs

```diff
@@ -44,12 +44,23 @@
   </InfinityLoader>
 {{/if}}
 
-<div class="mt-8">
+
+<div class="history-tl-container">
+  <ul class="tl">
   {{#each this.model.session as |session|}}
-    <Public::SessionItem @session={{session}} @timezone={{this.timezone}} />
+    <li class="tl-item" ng-repeat="item in retailer_history">
```

Comment:
Unnecessary code

Reply:
@iamareebjamal I have removed unrequired code from css and hbs both.